styleguide/source/assets/scss/04-templates/_three_column.scss

```diff
@@ -0,0 +1,68 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.three-column {
+  display: grid;
+  display: -ms-grid;
+  grid-template-columns: 1fr 2fr  1fr;
+  grid-template-rows: auto auto auto;
+  min-height: 30em;
+  -ms-grid-columns: 1fr 2fr 1fr;
+  -ms-grid-rows: auto auto auto;
+}
+
+.ama__page-content {
+  grid-column: 1 / 5;
+  grid-row: 1;
+
+  @media (min-width: $bp-small) {
+    grid-column: 1 / 4;
+    grid-row: 1 / 4;
+  }
+
+  @media (min-width: $bp-med) {
+    grid-column: 2;
+    grid-row: 1 / 4;
+    -ms-grid-column: 2;
+    -ms-grid-row: 1;
+  }
+}// end article
+
+
+.ama__sidebar-primary {
+  grid-column: 1 / 5;
+  grid-row: 2;
+  -ms-grid-row: 2;
```

Comment:
It looks like there are a lot of browser prefixes in here. Can you explain what these do? Did you need to add them manually?

Reply:
Unfortunately, IE11 does not understand standard css grid stuff.